app/src/main/java/ch/epfl/qedit/model/answer/AnswerFormat.java

```diff
@@ -19,6 +19,17 @@
      */
     private String text;
 
+    /** The correct answer */
+    private AnswerModel questionAnswer;
```

Comment:
I was a bit confused about the naming. Probably I would have picked `solution` but this is only a suggestion.

Reply:
ahaha Of course !